### PR TITLE
added definition of Ruby 2.2.2

### PR DESCRIPTION
**Pull request by nicolai86**

### CODE REVIEW

**Comment by DomT4** [comment]
:+1:. 2.1.6 and 2.0.0-p645 are available as well.

**Comment by nicolai86** [comment]
@DomT4 your right. Just added 2.1.6 and 2.0.0-p645. thanks

**Comment by philsturgeon** [comment]
How are we doing over here chaps?

**Comment by philsturgeon** [comment]
> Verifying checksum for http://cache.ruby-lang.org/pub/ruby/2.0/ruby-2.0.0-p645.tar.gz
> Error: http://cache.ruby-lang.org/pub/ruby/2.0/ruby-2.0.0-p645.tar.gz doesn't match its checksum 2dcdcf9900cb923a16d3662d067bc8c801997ac3e4a774775e387e883b3683e9

**Comment by DomT4** [comment]
Apologies, I opened a 2.0.0 [PR])(https://github.com/sstephenson/ruby-build/pull/740) and someone else got the 2.1.6 [one](https://github.com/sstephenson/ruby-build/pull/739), heh. I realised I could just do it myself instead of asking you to do so :smile_cat:. If maintainer wants to close mine in favour of yours, that's cool.

File: share/ruby-build/2.1.6 [hunk 1]
```diff
@@ -0,0 +1,2 @@
+install_package "openssl-1.0.1m" "https://www.openssl.org/source/openssl-1.0.1m.tar.gz#095f0b7b09116c0c5526422088058dc7e6e000aa14d22acca6a4e2babcdfef74" mac_openssl --if has_broken_mac_openssl
+install_package "ruby-2.1.6" "http://cache.ruby-lang.org/pub/ruby/2.1/ruby-2.1.6.tar.gz#7b5233be35a4a7fbd64923e42efb70b7bebd455d9d6f9d4001b3b3a6e0aa6ce9" ldflags_dirs standard verify_openssl
```
Comment (aripollak) [inline share/ruby-build/2.1.6]: This shows the correct hashes: https://www.ruby-lang.org/en/downloads/

**Comment by philsturgeon** [comment]
2.0.0-p645 had the wrong hashes. I've updated it here https://github.com/sstephenson/ruby-build/pull/741

**Comment by nicolai86** [comment]
cool. lets go with #741

**Pull request closed without merging**